pkg/hubble/parser/threefour/parser.go

```diff
@@ -201,6 +206,27 @@ func (p *Parser) resolveNames(epID uint32, ip net.IP) (names []string) {
 	return nil
 }
 
+func (p *Parser) resolvePolicies(fl *pb.Flow) (policies []string) {
+	if fl.Verdict != pb.Verdict_FORWARDED || fl.PolicyMatchType == monitorAPI.PolicyMatchNone {
+		return
+	}
+
+	npStore := p.storeGetter.GetK8sStore("networkpolicy")
+	nps := npStore.List()
```

Comment:
Do I understand correctly that we're querying k8s to retrieve all network policies for each Flow here?